common/app/views/fragments/discussionFooter.scala.html

```diff
@@ -8,8 +8,9 @@
         <div class="container__meta">
             <h2 class="container__meta__title">comments <span class="discussion__comment-count js-discussion-comment-count"></span></h2>
 
-            <p class="container__meta__item discussion__meta discussion__meta--open-signed-out"><a class="u-underline" href="@Configuration.id.url/signin?INTCMP=DOTCOM_COMMENTS_SIGNIN">Sign in</a>
-                or <a class="u-underline" href="@Configuration.id.url/register?INTCMP=DOTCOM_COMMENTS_REG">create your Guardian account</a> to join the discussion.
+            <p class="container__meta__item discussion__meta discussion__meta--open-signed-out">
+                <a class="u-underline" href="@Configuration.id.url/signin?INTCMP=DOTCOM_COMMENTS_SIGNIN@if(RegisterWithPhoneNumber.isSwitchedOn){&clientId=discussion"}>Sign in</a>
```

Comment:
the `clientId` in the other PR is `comments` not `discussion`

https://github.com/guardian/identity-frontend/pull/157/files#diff-852ecc2984d9c2bad5a4a7a160fde5a9R33